src/rsaidnumber/__init__.py

```diff
@@ -61,6 +62,11 @@ def parse(self):
             self.date_of_birth = datetime.strptime(
                 f"{year}{month}{day}", DATE_OF_BIRTH_FORMAT
             )
+
+            if self.date_of_birth > datetime.now():
+                self.date_of_birth = self.date_of_birth - relativedelta(
```

Comment:
Can we maybe use the built in `from datetime import timedelta` instead of adding a dependency on the `python-dateutil` package?

Reply:
Another possible solution might be:

```
now = datetime.now()
if self.date_of_birth.year > now.year:
    year = 1900 + self.date_of_birth.year % 1000
    self.date_of_birth.replace(year=year)
```
🤔 

this way you won't need any date/time arithmetic

Reply:
Only problem is this will stop working in 100 years 🤣

Reply:
> Can we maybe use the built in `from datetime import timedelta` instead of adding a dependency on the `python-dateutil` package?

Hol' up. It has timedelta builtin? Why when I looked this issue up people kept saying to use dateutil for it?

Reply:
The only issue I can see with using timedelta instead of this dateutil relativedelta is that timedelta doesn't support years, it goes up to weeks, so you end up having to use an approximation of days or weeks. So 365.2422 days in a year -> 36524.22 in 100, or 36525 when rounded.

With regards to the other solution, who knows if there will even be a South African id to worry about in 100 years haha.
I'm fine with either solution. What do you guys think?

Reply:
I'm going to push the timedelta change, but I can shift towards the arithmetic-less solution after, if you feel that's a better direction considering the 100 year flaw haha

Reply:
I am leaning towards an arithmetic solution because there is no need for an extra dependency